README.md

```diff
@@ -21,6 +21,10 @@ The API is documented in the [docs/api](./docs/api) directory of this repository
 The backend uses `OpenShift`'s built-in `OAuth` Server to authenticate users. To run it, configure environment variables in a new `.env` file:
 
 ```bash
+# Define the allowed origin regex pattern for WebSocket connections from the frontend. If not specified, all origins will be approved.
+ALLOWED_ORIGIN_REGEX="http:localhost:8080|https:example.com.*"
+
```

Comment:
Need to add this to the [hack script](https://github.com/dana-team/platform-backend/blob/main/hack/create-env.sh) and to the [ConfigMap](https://github.com/dana-team/platform-backend/blob/main/chart/templates/configmap.yaml) in the Helm chart (which needs some work in general)